lib/screens/v2/widget_instance/audio_only/content_summary.ex

```diff
@@ -27,8 +28,22 @@ defmodule Screens.V2.WidgetInstance.AudioOnly.ContentSummary do
   # end up adjacent to each other.
   @audio_sort_key_part [0]
 
+  def audio_serialize(
+        %__MODULE__{
+          screen: %Screen{
+            app_id: :pre_fare_v2,
+            app_params: %PreFare{departures: %ScreensConfig.Departures{sections: [_ | _]}}
+          }
+        } = t
+      ) do
+    %{
+      lines_at_station: t.lines_at_station,
+      showing_departures: true
```

Comment:
"Showing" is a slightly odd word to use in the context of audio, where we're not "showing" anything. How about `has_departures`?